### ISSUE TITLE
Reverse sorting of English and Persian  items 

### ISSUE THREAD

**Issue by Elfengour** [mamintoosi/persian-bib#1]
In references, Persian should be first. I am working with TeXlive 2020. 
![persianbib](https://user-images.githubusercontent.com/28195778/148934133-74d0dbe4-ef91-42cb-bbb3-bff2aa9682aa.png)

**Comment by Elfengour** [comment]
There is already aworkaround here [http://qa.parsilatex.com/29399/%D9%82%D8%B1%D8%A7%D8%B1-%DA%AF%D8%B1%D9%81%D8%AA%D9%86-%D9%85%D8%B1%D8%A7%D8%AC%D8%B9-%D9%81%D8%A7%D8%B1%D8%B3%DB%8C-%D8%AF%D8%B1-%D8%A7%D9%86%D8%AA%D9%87%D8%A7%DB%8C-%D9%85%D8%B1%D8%A7%D8%AC%D8%B9](url)

**Comment by mamintoosi** [comment]
Yes, I noticed that in page 4 of [Persian-bib userguide](https://ctan.org/pkg/persian-bib?lang=en).
